contracts/ERC20Safe.sol

```diff
@@ -48,9 +75,16 @@ contract ERC20Safe is BridgeRole, Pausable {
     function whitelistToken(
         address token,
         uint256 minimumAmount,
-        uint256 maximumAmount
+        uint256 maximumAmount,
+        bool mintBurn,
+        bool native
     ) external onlyAdmin {
+        if (!mintBurn) {
+            require(native, "Only native tokens can be stored!");
+        }
```

Comment:
What I understand is that you can store a `mintBurn` token or a `native` token, but a token can't be both at the same time.

However, this error message says "Only native tokens can be stored!", but the code clearly accepts `mintBurn` tokens also.

This check could be modified to:

```solidity
require(mintBurn || native, "Token must be either mintable/burnable or native!");
require(!(mintBurn && native), "Token cannot be both mintable/burnable and native!");
```